subscription/details.md

```diff
@@ -0,0 +1,51 @@
+
+Docker Pro includes all the features available in Personal, additionally, it includes unlimited private repositories, unlimited public repositories, unlimited [collaborators](../docker-hub/repos/access/index.md#collaborators-and-their-role) for public repositories, [Auto Builds](../docker-hub/builds/index.md) with 5 concurrent builds, 300 [Vulnerability Scans](../docker-hub/vulnerability-scanning.md), unlimited [Scoped Access Tokens](../docker-hub/access-tokens.md), and more.
+
+For a list of features available in each tier, see [Docker Pricing](https://www.docker.com/pricing/){: target="_blank" rel="noopener" class="_" id="dkr_docs_subscription_btl"}.
+
+## Docker Team
+
+**Docker Team** offers capabilities for collaboration, productivity, and security across organizations. It enables groups of developers to unlock the full power of collaboration and sharing combined with essential security features and team management capabilities.
+
+Docker Team includes everything included in Docker Pro, plus unlimited teams, [Auto Builds](../docker-hub/builds/index.md) with 15 concurrent builds, unlimited [Vulnerability Scanning](../docker-hub/vulnerability-scanning.md), advanced collaboration and management tools, including organization and team management with Role Based Access Control (RBAC) for the whole team, [audit logs](../docker-hub/audit-log.md), and more.
+
+For a list of features available in each tier, see [Docker Pricing](https://www.docker.com/pricing/){: target="_blank" rel="noopener" class="_" id="dkr_docs_subscription_btl"}.
+
+## Docker Business
+
+**Docker Business** offers centralized management and advanced security features for enterprises that use Docker at scale. It empowers leaders to manage their Docker development environments and speed up their secure software supply chain initiatives.
+
+In addition to all the features available in Docker Pro and Docker Team, Docker Business includes [Hardened Docker Desktop](../desktop/hardened-desktop/index.md) enhanced container isolation and settings management, centralized management, [Image Access Management](../docker-hub/image-access-management.md) with the ability to control what registries and content developers can access, ensuring teams are building secure from the start by only pulling trusted base images,[Single Sign-On](../single-sign-on/index.md),[System for Cross-domain Identity Management](../docker-hub/scim.md) and more.
+
+For a list of features available in each tier, see [Docker Pricing](https://www.docker.com/pricing/){: target="_blank" rel="noopener" class="_" id="dkr_docs_subscription_btl"}.
+
+### Self-serve
+
+A self-serve Docker Business subscription is where everything is set up by you. You can:
```

Comment:
Docker Business, as well as Docker Pro and Docker Team are all self-serve subscriptions.

While sales-assisted is only Docker Business. In other words, Docker Business subscription could be either self-serve or sales-assisted.